### PR TITLE
Revert Sequences scripting

### PR DESCRIPTION
**Pull request by hash-16**
### Description

Revert Support for SSMS scripting [PR](https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/1886)

### Test Scenarios Covered ###
* **Use case based -**

* **Boundary conditions -**

* **Arbitrary inputs -**

* **Negative test cases -**

* **Minor version upgrade tests -**

* **Major version upgrade tests -**

* **Performance tests -**

* **Tooling impact -**

* **Client tests -**

### Check List
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: test/JDBC/expected/babel_datatype_sqlvariant-vu-cleanup.out [hunk 1]
```diff
@@ -27,5 +25,9 @@ drop sequence babel_datatype_sqlvariant_vu_prepare_t2_sec;
 go
 drop sequence babel_datatype_sqlvariant_vu_prepare_t4_sec;
 go
+~~ERROR (Code: 3732)~~
+
+~~ERROR (Message: cannot drop sequence babel_datatype_sqlvariant_vu_prepare_t4_sec because other objects depend on it)~~
```
Comment (shalinilohia50) [inline test/JDBC/expected/babel_datatype_sqlvariant-vu-cleanup.out]: There is error while dropping sequence. Please fix this.
Why have you removed the line above?
```
drop table babel_datatype_sqlvariant_vu_prepare_t4;

File: test/JDBC/input/sql_variant/babel_datatype_sqlvariant-vu-cleanup.sql [hunk 2]
```diff
@@ -5,8 +5,6 @@ drop table babel_datatype_sqlvariant_vu_prepare_t2;
 go
 drop table babel_datatype_sqlvariant_vu_prepare_t3;
 go
-drop table babel_datatype_sqlvariant_vu_prepare_t4;
```
Comment (shalinilohia50) [inline test/JDBC/input/sql_variant/babel_datatype_sqlvariant-vu-cleanup.sql]: This change is not needed.

**Review by shalinilohia50 — APPROVED** [review]

File: test/JDBC/expected/babel_datatype_sqlvariant-vu-cleanup.out (hunk 1 above)
Comment (hash-16) [reply test/JDBC/expected/babel_datatype_sqlvariant-vu-cleanup.out]: Done

File: test/JDBC/input/sql_variant/babel_datatype_sqlvariant-vu-cleanup.sql (hunk 2 above)
Comment (hash-16) [reply test/JDBC/input/sql_variant/babel_datatype_sqlvariant-vu-cleanup.sql]: Done

**Review by shardgupta — APPROVED** [review]

**Pull request merged**